app/views/layouts/_cookie_banner.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render GovukComponent::CookieBannerComponent.new(
+  html_attributes: {
+    "data-module" => "govuk-cookie-banner",
+    "data-cookie-consent-name" => Settings.cookies.consent.name,
+    "data-cookie-consent-expiry-after-days" => Settings.cookies.consent.expire_after_days,
+  },
+) do |cookie_banner| %>
+  <%= cookie_banner.message(heading_text: t("cookies.heading"),
+                            text: t("cookies.message_html")) do |message| %>
+    <% message.action do %>
+      <button value="yes" type="button" name="cookies" class="govuk-button app-js-only" data-module="govuk-button">
+        <%= t("cookies.accept") %>
+      </button>
+      <button value="no" type="button" name="cookies" class="govuk-button app-js-only" data-module="govuk-button">
+        <%= t("cookies.reject") %>
+      </button>
+      <%= govuk_link_to(t("cookies.cookies_page_link")) %>
+    <% end %>
+  <% end %>
+<% end %>
+<%= render GovukComponent::CookieBannerComponent.new(html_attributes: {
+  "hidden" => true,
+  "data-module" => "govuk-cookie-after-consent-banner",
+}) do |cookie_banner| %>
+  <% cookie_preferences_link = govuk_link_to(t("cookies.after_consent.page_link_text")) %>
+  <%= cookie_banner.message(text: t("cookies.after_consent.message",
+                                    cookie_preferences_link: cookie_preferences_link).html_safe) do |message| %>
```

Comment:
I think we'll need to handle sending them to the right link with a bit of logic around the feature flags. While `google_analytics` is off we're still using the simple cookie page but if it's on we'd want to send them to the cookie preference page which would be enabled by the same flag

Reply:
This looks like it's been handled in the routes file itself - the feature flag will toggle the `cookies_path` between different routes.

Reply:
of course, i was the one who added it 🤦‍♂️